superbench/benchmarks/micro_benchmarks/disk_performance.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the Disk Performance benchmarks."""
+
+from pathlib import Path
+import json
+import os
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, ReturnCode
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class DiskPerformance(MicroBenchmarkWithInvoke):
+    """The disk performance benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'fio'
+
+        self.__io_patterns = ['seq', 'rand']
+        self.__io_types = ['read', 'write']
```

Comment:
Therefore, we cannot set sequence, random, read, and write?

Reply:
We have 4 kinds of tests: seq_read, seq_write, rand_read and rand_write, and we can enable each of them respectively

Reply:
For example, you can pass --seq_read_runtime=10 and --rand_read_runtime=10 at the same time.

Reply:
Got it.,